image_tools_py/test/test_showimage_cam2image.py.in

```diff
@@ -0,0 +1,102 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
```

Comment:
@clalancette this file uses legacy^2 `launch`, which IIRC is no longer available. It has to be updated.

Reply:
As we discussed, I've revamped this to use the new launch (but not launch_testing) infrastructure.  See 32b9dba for the implementation.